README.md

```diff
@@ -9,7 +9,7 @@ The website is generated using [Hugo Framework](https://gohugo.io/). To run the
 1. [Install Hugo](https://gohugo.io/getting-started/installing/)
 2. Run Hugo
     ```
-   ./hugo-start.sh
+   cargo run -- hugo
```

Comment:
I think we could also remove `hugo-start.sh` from the repo given that `cargo run -- hugo` executes the local Hugo binary.

https://github.com/infinyon/fluvio-website/blob/master/scripts/fluvio-cms/src/hugo/mod.rs#L12-L18

Reply:
We should if no longer used.

Reply:
I haven't removed it because it seems that the GitHub workflow is still using it https://github.com/search?q=user%3Ainfinyon+hugo-start&type=code